File: C++/interaccion usuario.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    // Variables para almacenar los datos del usuario
    string nombres, apellidos, profesion;
    int anioNacimiento, edad;

    // Solicitar datos al usuario
    cout << "Escriba sus nombres completos: ";
    getline(cin, nombres);  // Captura nombres con espacios

    cout << "Escriba sus apellidos completos: ";
    getline(cin, apellidos);  // Captura apellidos con espacios

    cout << "Escriba su profesión: ";
    getline(cin, profesion);  // Captura profesión con espacios

    cout << "Escriba su año de nacimiento: ";
    cin >> anioNacimiento;

    // Calcular edad
    edad = 2025 - anioNacimiento;

    // Mostrar resultado
    cout << "El (La) " << profesion << " " << nombres << " " << apellidos << " tiene " << edad << " años." << endl;

    return 0;
}

File: JAVA/interaccion usuario.java
import java.util.Scanner;

public class DatosUsuario {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Solicitar los datos al usuario
        System.out.print("Escriba sus nombres completos: ");
        String nombres = scanner.nextLine();

        System.out.print("Escriba sus Apellidos completos: ");
        String apellidos = scanner.nextLine();

        System.out.print("Escriba su profesión: ");
        String profesion = scanner.nextLine();

        System.out.print("Escriba su año de nacimiento: ");
        int añoNacimiento = scanner.nextInt();

        // Calcular la edad
        int edad = 2025 - añoNacimiento;

        // Mostrar el resultado
        System.out.println("El (La) " + profesion + " " + nombres + " " + apellidos + " tiene " + edad + " años.");

        scanner.close(); // Cerrar el scanner
    }
}
